syntax = "proto3"; 

package register_proto;

message user_register_request
{
    string account=1;
    string password=2;
    string name=3;
}

message user_register_reply
{
    bool issuccess=1;
}